hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -782,6 +781,10 @@ private void nullOutSideEffectFields() {
 			assessedCustomFees = NO_CUSTOM_FEES;
 			newTokenAssociations = NO_NEW_TOKEN_ASSOCIATIONS;
 			alias = MISSING_ALIAS;
+			/*- if this is a revert of a child record we want to have contractCallResult -*/
+			if (!isRevertOfChildRecord) {
```

Comment:
Did intellij post a warning about always inverting isRevertOfChildRecord?  Would it work better to flip it and have the parameter be `removeCallResult`?